deploy/role.yaml

```diff
@@ -17,6 +17,18 @@ rules:
   - secrets
   verbs:
   - '*'
+- apiGroups:
+  - "route.openshift.io"
```

Comment:
The permissions for this group are already in this file further up.

Reply:
modified.